packages/suite/src/hooks/wallet/form/useCompose.ts

```diff
@@ -231,8 +236,10 @@ export const useCompose = <TFieldValues extends FormState>({
             : undefined;
         if (composedTx && composedTx.type === 'final') {
             // sign workflow in Actions:
-            // signTransaction > sign[COIN]Transaction > requestPushTransaction (modal with promise decision)
-            const result = await dispatch(signTransaction(values, composedTx));
+            // signTransactionThunk > sign[COIN]Transaction > requestPushTransaction (modal with promise decision)
```

Comment:
Remove or update. Now there are no functions with this name

Reply:
fixed by c2606f3977